src/index.js

```diff
@@ -39,7 +39,26 @@ const create = (privateKey, value, seq, lifetime, callback) => {
   // Validity in ISOString with nanoseconds precision and validity type EOL
   const isoValidity = nanoDateEol.toISOStringFull()
   const validityType = ipnsEntryProto.ValidityType.EOL
+  _create(privateKey, value, seq, isoValidity, validityType, callback)
+}
+
+/**
+ * Same as create(), but prevent generating a Date().
+ * WARNING: nano precision is not standard, make sure the value in seconds is 9 orders of magnitude lesser than the one provided.
+ * @param {Object} privateKey private key for signing the record.
+ * @param {string} value value to be stored in the record.
+ * @param {number} seq number representing the current version of the record.
+ * @param {string} expiration lifetime of the record (in nanoseconds).
```

Comment:
Update the description of the expiration to something `expiration time of the record (in nanoseconds)`